Settings Page layout changes

infinite-persistence opened the pull request:
## Test
- https://kp.odysee.com

## Issue
- Closes [#6363 split settings into separate subareas with navigation](https://github.com/lbryio/lbry-desktop/issues/6363)
- Closes [#6395 Advanced settings not persisting ](https://github.com/lbryio/lbry-desktop/issues/6395)
    - "Advanced settings" page no longer exists.

## Comments
- I left the individual commits unsquashed to hopefully help the review.  Probably easiest to go through each commit one-by-one.
- For medium and small screen layout, the navigation is hidden for now.  I think this is fine since we are now showing all settings (scroll) and the total settings is quite small.  If needed, we can probably display the navigation as a Tab at the top for these layouts.

## Debt
- <strike>"Creator Settings" remained in the old style. Will update the style together in an upcoming PR for "min tips". </strike>

## Notes
https://www.notion.so/lbry/Settings-Page-Debt-d07a232a03fa469e8bae47ce2bf9e307

infinite-persistence commented:
### 2cd691d to 2b53c74 (ready for review)
Notes:  https://www.notion.so/lbry/Settings-Page-Debt-d07a232a03fa469e8bae47ce2bf9e307

tzarebczan commented:
- [ ] the drop-down carrot seems too close to the right side of the box
- [ ] This page looks a bit strange to me with some of the grouped settings (tip) and dual pane (Similar to the other dual pane on system settings). Maybe if we added help test under each category and made the Tip heading > "Tipping with Comments" to fill in the heading (+ some simple help text). 
![image](https://user-images.githubusercontent.com/8120721/130142150-da835b63-f34d-41d0-967d-ec462116df3e.png)

- [ ] Bank accounts listed twice

- [ ] Center all pages? This should also say "Add a card" - the heading changes. 
![image](https://user-images.githubusercontent.com/8120721/130142405-195c190c-dc30-49e3-a9ae-b7102588f53c.png)
![image](https://user-images.githubusercontent.com/8120721/130142427-030274fe-84e6-4c16-9814-32ebde8a5b67.png)

- [ ] navigation steps count when switching settings pages, so the Back behavior is strange - goes back through each. Back should always go back to the previous non settings page right?

infinite-persistence commented on static/app-strings.json:
```diff
@@ -1744,8 +1746,9 @@
   "Add moderator": "Add moderator",
   "Enter a @username or URL": "Enter a @username or URL",
-  "examples: @channel, @channel#3, https://odysee.com/@Odysee:8, lbry://@Odysee#8": "examples: @channel, @channel#3, https://odysee.com/@Odysee:8, lbry://@Odysee#8",
+  "Enter a channel name or URL to add as a moderator.\nExamples:\n - @channel\n - @channel#3\n - https://odysee.com/@Odysee:8\n - lbry://@Odysee#8": "Enter a channel name or URL to add as a moderator.\nExamples:\n - @channel\n - @channel#3\n - https://odysee.com/@Odysee:8\n - lbry://@Odysee#8",
```
Technically supported in Transifex
![image](https://user-images.githubusercontent.com/64950861/130304961-fbc8f949-2c0c-4a1d-ba04-97b48f94e1cc.png)

infinite-persistence commented:
### 2b53c74 to fc4de99 
- Changed the "Back" button behavior from link-style (ala Chrome) to stack style.
- Removed double "Bank Accounts" (merge error).
- Moved "Homepages" to the top in [Appearance].
- Switched from multi-pane to mult-row style per Tom's feedback.
    - The multi-pane version aimed to provide visual separation between label and value (left is always label, right is always value), but agree that it could be too clamped for radio buttons.
    - <img src="https://user-images.githubusercontent.com/64950861/130352866-e82b0830-2285-4243-8bb4-f62d39caa8b6.png" width="600">
- Added extra top-margin when navigating to a sub-section.
